Assets/MRTK/Core/Providers/InputAnimation/IMixedRealityInputRecordingService.cs

```diff
@@ -63,6 +62,24 @@ public interface IMixedRealityInputRecordingService : IMixedRealityInputDeviceMa
         /// <param name="filename">Name of the file to create.</param>
         /// <param name="directory">Directory in which to create the file. If null the persistent data path of the app is used.</param>
         /// <returns>File path where input has been recorded.</returns>
-        string SaveInputAnimation(string filename, string directory = null);
+        string SaveInputAnimation(string filename, string directory);
```

Comment:
Why not allow the directory to continue to be `null`? The param doc above states

>If null the persistent data path of the app is used.

Re-adding `= null` would also help remove a breaking change.

Reply:
That default value is redundant since inputting a single string will call the other overload